tests/phpunit/CRM/Contribute/ActionMapping/ByTypeTest.php

```diff
@@ -232,9 +232,9 @@ public function scheduleForSoftCreditor() {
   }
 
   public function useHelloFirstNameStatus() {
-    $this->schedule->subject = 'Hello, {contact.first_name}. @{contribution.status}. (via subject)';
-    $this->schedule->body_html = '<p>Hello, {contact.first_name}. @{contribution.status}. (via body_html)</p>';
-    $this->schedule->body_text = 'Hello, {contact.first_name}. @{contribution.status} (via body_text)';
+    $this->schedule->subject = 'Hello, {contact.first_name}. @{contribution.contribution_status_id:name}. (via subject)';
```

Comment:
@eileenmcnaughton shouldn't this shift to be label?

Reply:
@seamuslee001 yeah - I can do - the follow up patches I have sitting around use a combo of name & label for the various fields

Reply:
@seamuslee001 OK - I've added specific tests for label into the PR - I originally considered removing the status token from this line but left it in - however it's just an extra place that token is tested rather than anything meaningful